dev_docs/contributing/third_party_dependencies.mdx

```diff
@@ -67,6 +67,57 @@ on Github:
   [Permitted Open Source Licenses list](https://github.com/elastic/open-source/blob/main/elastic-product-policy.md#permitted-licenses-list).
 </DocCallOut>
 
+### Dependency evaluation
+
+1. **Snyk health check**
+    - Ensure a minimum health score is >= 80 in [Snyk Advisor](https://snyk.io/advisor).
+    - If the health score is below 80, a valid business justification must be provided.
+
+2. **Vulnerability assessment**. Check vulnerability reports from Snyk for the exact dependency version, available through [Snyk Advisor](https://snyk.io/advisor).
+    If vulnerabilities are present (production dependencies only):
+
+    - **Critical (CVSS 9.0 - 10.0)**:
+      - **False positive:** Proceed to Step 3.
+      - **Confirmed vulnerability:** Reject dependency.
+
+    - **High (CVSS 7.0 - 8.9)**:
+      - **False positive:** Proceed to Step 3.
+      - **Confirmed vulnerability:** Reject dependency.
+
+    - **Medium/Low (CVSS 0.0 - 6.9)**:
+      - **False positive:** Proceed to Step 3.
+      - **Confirmed vulnerability:** Exception can be granted if the vulnerability is confirmed to be impossible to exploit.
```

Comment:
I'd say "Exception can be granted if the vulnerability is confirmed to be impossible to exploit -->and there is no better alternative<---” because in such case we'd like the PR author to consider alternatives, since something that cannot be exploited today might be exploitable in the future.